substra/sdk/schemas.py

```diff
@@ -155,6 +155,23 @@ def exclusive_paths(cls, values):  # noqa: N805
             raise ValueError("'path' or 'paths' field must be set.")
         return values
 
+    @pydantic.model_validator(mode="before")
+    def resolve_paths(cls, values):  # noqa: N805
```

Comment:
No typing on `values`?

Reply:
I just copy paste it from abose. We might add it though, but it's `values` is provided by pydantic so it's not that bad no ? Wdyt ?

Reply:
I would rather use `values: Any): -> Any` or create a custom type (e.g. `ValueType`) and use `values: ValueType): -> ValueType`